<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1444</width>
    <height>742</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetMinimumSize</enum>
      </property>
      <item>
       <widget class="GLWidget" name="displayWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1444</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
   </widget>
   <addaction name="menuView"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="optionsDockWidget">
   <property name="minimumSize">
    <size>
     <width>766</width>
     <height>360</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>Options</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_12">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="tabPosition">
        <enum>QTabWidget::West</enum>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="connectionTab">
        <attribute name="title">
         <string>Connection</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_11">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLineEdit" name="portNameLineEdit"/>
           </item>
           <item>
            <widget class="QComboBox" name="portBaudRateComboBox"/>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="startPushButton">
           <property name="text">
            <string>start</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="drawingTab">
        <attribute name="title">
         <string>Drawing</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QCheckBox" name="drawStickmanCheckBox">
           <property name="text">
            <string>Draw stickman</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Debug drawer debug level:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QCheckBox" name="drawRightMarkerCheckBox">
           <property name="text">
            <string>Draw right marker</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="spinBox">
           <property name="value">
            <number>1</number>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QCheckBox" name="drawLeftMarkerCheckBox">
           <property name="text">
            <string>Draw left marker</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QPushButton" name="resetBoxesPushButton">
           <property name="text">
            <string>Reset boxes</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QCheckBox" name="drawBoxesCheckBox">
           <property name="text">
            <string>Draw boxes</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="calibrationTab">
        <attribute name="title">
         <string>Calibration</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QGroupBox" name="leftGroupBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="title">
            <string>Left</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QTabWidget" name="leftTabWidget">
              <property name="currentIndex">
               <number>0</number>
              </property>
              <widget class="QWidget" name="leftRegularTab">
               <attribute name="title">
                <string>Regular</string>
               </attribute>
               <layout class="QVBoxLayout" name="verticalLayout_7">
                <item>
                 <layout class="QHBoxLayout" name="leftNearHorizontalLayout">
                  <item>
                   <widget class="QPushButton" name="leftNearPushButton">
                    <property name="text">
                     <string>get near</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLCDNumber" name="leftNearXLcdNumber"/>
                  </item>
                  <item>
                   <widget class="QLCDNumber" name="leftNearYLcdNumber"/>
                  </item>
                  <item>
                   <widget class="QLCDNumber" name="leftNearZLcdNumber"/>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="leftFarHorizontalLayout">
                  <item>
                   <widget class="QPushButton" name="leftFarPushButton">
                    <property name="text">
                     <string>get far</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLCDNumber" name="leftFarXLcdNumber"/>
                  </item>
                  <item>
                   <widget class="QLCDNumber" name="leftFarYLcdNumber"/>
                  </item>
                  <item>
                   <widget class="QLCDNumber" name="leftFarZLcdNumber"/>
                  </item>
                 </layout>
                </item>
                <item>
                 <widget class="QPushButton" name="leftRegularCalibratePushButton">
                  <property name="text">
                   <string>Calibrate</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="leftExperimentalTab">
               <attribute name="title">
                <string>Experimental</string>
               </attribute>
               <layout class="QVBoxLayout" name="verticalLayout_8">
                <item>
                 <widget class="QPushButton" name="leftFrontPushButton">
                  <property name="text">
                   <string>Left Front</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="leftLeftPushButton">
                  <property name="text">
                   <string>Left Left</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="leftZeroPushButton">
                  <property name="text">
                   <string>Left Zero</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="leftExperimentalCalibratePushButton">
                  <property name="text">
                   <string>Calibrate</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="leftResetPushButton">
              <property name="text">
               <string>Reset</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="rightGroupBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="title">
            <string>Right</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QTabWidget" name="rightTabWidget">
              <property name="currentIndex">
               <number>0</number>
              </property>
              <widget class="QWidget" name="rightRegularTab">
               <attribute name="title">
                <string>Regular</string>
               </attribute>
               <layout class="QVBoxLayout" name="verticalLayout_5">
                <item>
                 <layout class="QHBoxLayout" name="rightNearHorizontalLayout">
                  <item>
                   <widget class="QPushButton" name="rightNearPushButton">
                    <property name="text">
                     <string>get near</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLCDNumber" name="rightNearXLcdNumber"/>
                  </item>
                  <item>
                   <widget class="QLCDNumber" name="rightNearYLcdNumber"/>
                  </item>
                  <item>
                   <widget class="QLCDNumber" name="rightNearZLcdNumber"/>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="rightFarHorizontalLayout">
                  <item>
                   <widget class="QPushButton" name="rightFarPushButton">
                    <property name="text">
                     <string>get far</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLCDNumber" name="rightFarXLcdNumber"/>
                  </item>
                  <item>
                   <widget class="QLCDNumber" name="rightFarYLcdNumber"/>
                  </item>
                  <item>
                   <widget class="QLCDNumber" name="rightFarZLcdNumber"/>
                  </item>
                 </layout>
                </item>
                <item>
                 <widget class="QPushButton" name="rightRegularCalibratePushButton">
                  <property name="text">
                   <string>Calibrate</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="rightExperimentalTab">
               <attribute name="title">
                <string>Experimental</string>
               </attribute>
               <layout class="QVBoxLayout" name="verticalLayout_6">
                <item>
                 <widget class="QPushButton" name="rightFrontPushButton">
                  <property name="text">
                   <string>Right Front</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="rightRightPushButton">
                  <property name="text">
                   <string>Right Right</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="rightZeroPushButton">
                  <property name="text">
                   <string>Right Zero</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="rightExperimentalCalibratePushButton">
                  <property name="text">
                   <string>Calibrate</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="rightResetPushButton">
              <property name="text">
               <string>Reset</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="debugTab">
        <attribute name="title">
         <string>Debug</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_12">
         <item>
          <widget class="QCheckBox" name="debugEnableCheckBox">
           <property name="toolTip">
            <string>Log debug data</string>
           </property>
           <property name="text">
            <string>Debug</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="debugIntervalSpinBox">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="toolTip">
            <string>Show sample data every ... samples</string>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="value">
            <number>20</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="benchmarkPushButton">
           <property name="text">
            <string>Benchmark</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GLWidget</class>
   <extends>QWidget</extends>
   <header>glwidget.h</header>
   <container>1</container>
   <slots>
    <slot>toggleDebugEnable(bool)</slot>
    <slot>setDebugInterval(int)</slot>
    <slot>setDrawStickman(bool)</slot>
    <slot>setDrawRightMarker(bool)</slot>
    <slot>setDrawLeftMarker(bool)</slot>
    <slot>setDebugLevel(int)</slot>
    <slot>resetBoxes()</slot>
    <slot>setDrawBoxes(bool)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>debugEnableCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>debugIntervalSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>262</x>
     <y>704</y>
    </hint>
    <hint type="destinationlabel">
     <x>520</x>
     <y>707</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>debugEnableCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>displayWidget</receiver>
   <slot>toggleDebugEnable(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>140</x>
     <y>704</y>
    </hint>
    <hint type="destinationlabel">
     <x>155</x>
     <y>95</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>debugIntervalSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>displayWidget</receiver>
   <slot>setDebugInterval(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>520</x>
     <y>707</y>
    </hint>
    <hint type="destinationlabel">
     <x>525</x>
     <y>98</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>drawStickmanCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>displayWidget</receiver>
   <slot>setDrawStickman(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>130</x>
     <y>568</y>
    </hint>
    <hint type="destinationlabel">
     <x>263</x>
     <y>109</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>drawRightMarkerCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>displayWidget</receiver>
   <slot>setDrawRightMarker(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>116</x>
     <y>598</y>
    </hint>
    <hint type="destinationlabel">
     <x>79</x>
     <y>114</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>drawLeftMarkerCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>displayWidget</receiver>
   <slot>setDrawLeftMarker(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>168</x>
     <y>631</y>
    </hint>
    <hint type="destinationlabel">
     <x>126</x>
     <y>91</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>displayWidget</receiver>
   <slot>setDebugLevel(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>771</x>
     <y>601</y>
    </hint>
    <hint type="destinationlabel">
     <x>346</x>
     <y>99</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>resetBoxesPushButton</sender>
   <signal>clicked()</signal>
   <receiver>displayWidget</receiver>
   <slot>resetBoxes()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>771</x>
     <y>634</y>
    </hint>
    <hint type="destinationlabel">
     <x>609</x>
     <y>188</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>drawBoxesCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>displayWidget</receiver>
   <slot>setDrawBoxes(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>204</x>
     <y>652</y>
    </hint>
    <hint type="destinationlabel">
     <x>203</x>
     <y>141</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
